src/components/drawer/drawer.stories.mdx

```diff
@@ -39,6 +46,136 @@ import Drawer from 'carbon-react/lib/components/drawer';
   </Story>
 </Preview>
 
+### Various content:
```

Comment:
I think that maybe better will be to have two examples with FlatTable component. 
One as a content of Drawer and one as a content of the sidebar. It will make the examples more clear

Reply:
updated